pkg/client/restconfig/config.go

```diff
@@ -55,16 +55,6 @@ func newConfigPath() (string, error) {
 	return path, nil
 }
 
-// newConfigFromPath creates a rest.Config from a configuration file at the
-// supplied path.
-func newConfigFromPath(path string) (*rest.Config, error) {
```

Comment:
What's the motivation for moving these functions to the other file rather than renaming them here?

Reply:
The other file has the public constructors. But it doesn’t really matter.

Reply:
Having them next to where they’re used makes for less flipping between files when reading the code. 

If you prefer them here, tho. I can move them back. I’m not really sure why these are two separate files.